Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -200,7 +239,21 @@ public struct TLSConfiguration {
 
     /// The pre-TLS1.3 cipher suites supported by this handler. This uses the OpenSSL cipher string format.
     /// TLS 1.3 cipher suites cannot be configured.
-    public var cipherSuites: String
+    public var cipherSuites: String = defaultCipherSuites
+    
+    /// Public property used to set the internal cipherSuiteFormattedString from NIOTLSCipher.
+    public var cipherSuite: [NIOTLSCipher] {
+        get {
+            guard let sslConext = try? NIOSSLContext(configuration: self) else {
+                return []
+            }
+            return sslConext.cipherSuites
+        }
+        set {
+            let assignedCiphers = newValue.map { $0.standardName }
+            cipherSuites = assignedCiphers.joined(separator: ":")
```

Comment:
Nit: we use explicit `self` in SwiftNIO where we can.

Reply:
Resolved in https://github.com/apple/swift-nio-ssl/pull/293/commits/dc9625669fc700a8f1cb8a064ad4c59a84fc6563